Makefile

```diff
@@ -24,9 +23,6 @@ HELMNAME = IMC
 
 default: build test
 
-install:
```

Comment:
To install dependencies, we need to have **install** target with `go mod download`